spec/models/i14y_search_spec.rb

```diff
@@ -1,11 +1,39 @@
+# frozen_string_literal: true
+
 require 'spec_helper'
 
 describe I14ySearch do
-  fixtures :affiliates, :i14y_drawers, :i14y_memberships, :tag_filters
-
   let(:affiliate) { affiliates(:i14y_affiliate) }
   let(:i14y_search) { described_class.new(affiliate: affiliate, query: 'marketplase') }
 
+  describe '#initialize' do
```

Comment:
Primary changes are here and beginning at line 76.  Remaining changes rubocop appeasement.